dev/src/components/forms/CourseForm.tsx

```diff
@@ -0,0 +1,156 @@
+                        <div className="input-group-text">Organization</div>
+                    </div>
+                    <input className="form-control" disabled={orgFound ? true : false} onKeyUp={e => setOrgName(e.currentTarget.value)}></input>
+                    <span className={orgFound ? "btn btn-success disabled" : "btn btn-primary"} onClick={!orgFound ? () => getOrganization() : () => { return }}>
+                        {orgFound ? <i className="fa fa-check"></i> : "Find"}
+                    </span>
+                </div>
+            </div>
+            {orgFound &&
+                <form className="form-group" onSubmit={async e => await submitHandler(e)}>
+                    <div className="row">
+                        <FormInput prepend="Name"
+                            name="courseName"
+                            placeholder={"Course Name"}
+                            defaultValue={editCourse?.getName()}
+                            onChange={handleChange}
+                        />
+                    </div>
+                    <div className="row">
+                        <FormInput
+                            prepend="Code"
+                            name="courseCode"
+                            placeholder={"(ex. DAT320)"}
+                            defaultValue={editCourse?.getCode()}
+                            onChange={handleChange}
+                        />
+                        <FormInput
+                            prepend="Tag"
+                            name="courseTag"
+                            placeholder={"(ex. Fall / Spring)"}
+                            defaultValue={editCourse ? editCourse.getTag() : defaultTag(date)}
+                            onChange={handleChange}
+                        />
+                    </div>
+                    <div className="row">
+                        <FormInput
+                            prepend="Slip days"
+                            name="slipDays"
+                            placeholder={"(ex. 7)"}
+                            defaultValue={editCourse?.getSlipdays().toString()}
+                            onChange={handleChange}
+                            type="number"
+                        />
+                        <FormInput
+                            prepend="Year"
+                            name="courseYear"
+                            placeholder={"(ex. 2021)"}
+                            defaultValue={editCourse ? editCourse.getYear().toString() : defaultYear(date).toString()}
+                            onChange={handleChange}
+                            type="number"
+                        />
+                    </div>
+                    <input className="btn btn-primary" type="submit" value={editCourse ? "Edit Course" : "Create Course"} />
+                </form>
+            }
+        </div>
+
+    )
+}
+
+export default CourseForm
+
+// Helper functions for default values for new courses
```

Comment:
Move these helpers to other file or above so that the file ends with the export default.